### ISSUE TITLE
Add a confirmation prompt to /clearobjects and /clearobjects quick

### ISSUE THREAD

**Issue by IhrFussel** [minetest/minetest#6918]
It already happened to me a lot of times that I accidentally typed /clearobjects instead of //clearobjects (command from World Edit to remove objects in a marked region) and had all objects on the server deleted.

I suggest to add a prompt (chat message) that asks the user if they really want to proceed with this action and that it might take a long time (in full mode).

If server owners run those commands by accident (for whatever reason), A LOT of progress could get lost depending on the game type and mods).

EDIT: Removed formspec as prompt option since a chat message is more than enough.

**Comment by Ezhh** [comment]
I think chat message would make most sense (WE already shows a good example of confirmation messages for large scale actions). There is no need for a formspec here.

**Comment by red-001** [comment]
better chat message would be better since it means you can still run it without login in with a minetest client (e.g. irc_commands)

**Comment by benrob0329** [comment]
What about a general /y and /n commands with a ```minetest.chat_conform(player)``` for mods?

**Comment by IhrFussel** [comment]
@benrob0329 that's another issue.

I'm talking about adding confirmation prompts to critical builtin commands and not to add them to mods.

**Comment by beyondlimits** [comment]
`/clearobjects`
Say a magic word.
`/clearobjects please`

**Comment by paramat** [comment]
Agreed it's WE's fault and it should change, please contact the WE maintainers. However i'm neutral on a confirmation.

**Comment by paramat** [comment]
1 year bump, any core dev support?
